.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: node_js
 sudo: 'false'
-before_script: npm install -g polymer-cli
+before_script:
+  - npm install -g polymer-cli
+  - >-
```

Comment:
add `git checkout package-lock.json`

Reply:
(to prevent travis from getting upset about arbitrary changes it might make to package-lock.json, e.g. when npm updates)

Reply:
I've updated the [gist](https://gist.github.com/e111077/7e2713e3777f650987f76036cd184767#file-travis-yml-L5)

Reply:
Added.